test/conformance/testing/include/uur/fixtures.h

```diff
@@ -696,8 +696,8 @@ struct urKernelExecutionTest : urKernelTest {
                                               sizeof(zero), 0, size, 0, nullptr,
                                               nullptr));
         ASSERT_SUCCESS(urQueueFinish(queue));
-        ASSERT_SUCCESS(
-            urKernelSetArgMemObj(kernel, current_arg_index, mem_handle));
+        ASSERT_SUCCESS(urKernelSetArgMemObj(kernel, current_arg_index,
+                                            mem_handle, nullptr));
```

Comment:
```suggestion
                                            nullptr, mem_handle));
```

Reply:
it was fixed here https://github.com/oneapi-src/unified-runtime/pull/571/commits/87062026b61250c722f2bb9c0f738226f207543a